qiskit_experiments/library/hamiltonian/heat_base.py

```diff
@@ -0,0 +1,243 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Base Class for general Hamiltonian Error Amplifying Tomography experiments.
+"""
+
+from abc import ABC
+from typing import List, Tuple, Optional
+
+from qiskit import circuit, QuantumCircuit
+from qiskit.providers import Backend
+
+from qiskit_experiments.framework import BaseExperiment, BatchExperiment, Options
+from qiskit_experiments.curve_analysis import ParameterRepr
+from .heat_analysis import HeatElementAnalysis, HeatAnalysis
+
+
+class HeatElement(BaseExperiment):
```

Comment:
This can be instantiated. User can easily test custom amplification sequence, e.g. for research, by directly providing circuit and instantiate it. This is because we cannot maintain every possible sequence pattern in experiments.